samples/seh/main.c

```diff
@@ -0,0 +1,44 @@
+#include <hal/debug.h>
+#include <hal/video.h>
+#include <windows.h>
+
+// The nullpointer deref is in its own function, because LLVM can't catch
```

Comment:
nit: nxdk users might not be aware what LLVM is or that nxdk uses it. I feel like this should be mentioned. Potentially mention "clang" instead, which is already slightly better.